network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -130,6 +134,36 @@ public void visitGenerator(Generator generator) {
         }
     }
 
+    @Test
+    public void svcTest() {
+        try (NetworkStoreService service = new NetworkStoreService(getBaseUrl())) {
```

Comment:
That would be more robust if after the network creation + flush, you close the  NetworkStoreService and reopen one to read and assert data (so you will have 2 try() {} sections).

Reply:
I agree. Done.